components/mailgun/sources/new-log-data/new-log-data.js

```diff
@@ -0,0 +1,90 @@
+const {
+  props,
+  methods,
+  ...common
+} = require("../common/timer-based");
+
+module.exports = {
+  ...common,
+  key: "mailgun-new-log-data",
+  name: "New Log Data",
+  type: "source",
+  description: "Emit new event when new data is logged in Mailgun's Control Panel. Occurs for " +
+    "most actions within the associated Mailgun account.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...props,
+  },
+  methods: {
+    ...methods,
+    generateMeta(payload) {
+      return {
+        id: payload.id,
+        summary: `${payload.timestamp} [${payload["log-level"]}] ${payload.event}`,
```

Comment:
**Suggestion**
Remove `payload.timestamp` from this `summary` so more of the `event` is visible in the UI.
<img width="254" alt="log-data" src="https://user-images.githubusercontent.com/19861096/155042832-7fbad68a-addf-4ee2-921c-5df1feb8e812.png">
E.g.,
```js
        summary: `[${payload["log-level"]}] ${payload.event}`,
```